playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -37,12 +39,14 @@
       Name: "{{ item.0 }}"
 
 - set_fact:
-    instance_groups: tag_created-by_{{ created_by }}, tag_env_{{ env }}, tag_host-type_{{ host_type }}, tag_env-host-type_{{ env_host_type }}
+    instance_groups: tag_created-by_{{ created_by }}, tag_env_{{ env }}, tag_host-type_{{ host_type }}, tag_env-host-type_{{ env_host_type }}, tag_deployment-type_{{ deployment_type }}
```

Comment:
We have to be extremely careful here. AWS only gives us 10 tag slots for instances. We're now taking up 5 of them by default.

I don't think deployment type rises to the level that it should take one of those slots.

If you agree, I'm ok with this being fixed in a separate PR.